autopts/wid/mics.py

```diff
@@ -0,0 +1,103 @@
+#
+# auto-pts - The Bluetooth PTS Automation Framework
+#
+# Copyright (c) 2023, Codecoup.
+#
+# This program is free software; you can redistribute it and/or modify it
+# under the terms and conditions of the GNU General Public License,
+# version 2, as published by the Free Software Foundation.
+#
+# This program is distributed in the hope it will be useful, but WITHOUT
+# ANY WARRANTY; without even the implied warranty of MERCHANTABILITY or
+# FITNESS FOR A PARTICULAR PURPOSE.  See the GNU General Public License for
+# more details.
+#
+
+import logging
+import sys
+
+from autopts.ptsprojects.stack import get_stack
+from autopts.pybtp import btp
+from autopts.pybtp import defs
+from autopts.pybtp.types import WIDParams
+
+log = logging.debug
+mute_state = None
```

Comment:
I recently discovered that it is better to make any global variable part of the stack if it is to be cleared after each test case. This way we will ensure that it is always cleared.
```suggestion
```